src/Symfony/Bundle/FrameworkBundle/Command/DebugAutowiringCommand.php

```diff
@@ -104,7 +104,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $serviceLine = sprintf('<fg=yellow>%s</>', $serviceId);
             if ($builder->hasAlias($serviceId)) {
                 $hasAlias[$serviceId] = true;
-                $serviceLine .= ' <fg=cyan>('.$builder->getAlias($serviceId).')</>';
+                $serviceAlias = $builder->getAlias($serviceId);
+                $serviceLine .= ' <fg=cyan>('.$serviceAlias.')</>';
+
+                if ($serviceAlias->isDeprecated()) {
```

Comment:
since `Alias::isDeprecated()` was added in master, we need to bump the minimum version of DI in FWB's composer.json to `^4.3`

Reply:
i'll bump it to `^4.3`